src/renderer.rs

```diff
@@ -267,11 +267,11 @@ fn version_footer() -> Markup {
     }
 }
 
-fn wget_footer(title_path: &str, current_user: Option<&CurrentUser>) -> Markup {
+fn wget_footer(abs_path: &str, current_user: Option<&CurrentUser>) -> Markup {
     let count = {
-        let count_slashes = title_path.matches('/').count();
-        if count_slashes > 0 {
-            count_slashes - 1
+        let count_slashes = abs_path.matches('/').count();
+        if count_slashes >= 4 {
+            count_slashes - 4
```

Comment:
That's a bit sketchy. Why 4?

Reply:
Oh, that change is because `abs_path` has more slashes than the breadcrumbs, since it includes the protocol. But wait, why did I increase it by three, not just two? It should be `http://test.net/my-folder/.git` compared to `test.net/my-folder/.git`, only two more slashes... I made it so the `--cut-dirs` flag has the same value as before the change, but I can't explain exactly why it is that way, Imma investigate

Reply:
Oh I know why! `abs_path` includes an extra slash at the end which the breadcrumbs didn't, so the two slashes from the protocol plus the one at the end made me increase it by three. But that means that there is now two slashes at the end of the path in the command, I need to fix that.

Reply:
I think I made the fix and it should be part of the PR now, I made an embarrassing typo in the commit message though which I discovered too late because I wasn't careful before pushing to my fork, I fixed the typo in an amend and force pushed that to my fork, I hope that's not causing any issues.